bin/elixir.ps1

```diff
@@ -0,0 +1,364 @@
+      $erlangParams += $allArgs[++$i]
+      break
+    }
+
+    "--vm-args" {
+      $erlangParams += "-args_file"
+      $erlangParams += $allArgs[++$i]
+      break
+    }
+
+    "--logger-otp-reports" {
+      $private:tempVal = $allArgs[$i + 1]
+      if ($tempVal -in @("true", "false")) {
+        $erlangParams += @("-logger", "handle_otp_reports", $allArgs[++$i])
+      }
+      break
+    }
+
+    "--logger-sasl-reports" {
+      $private:tempVal = $allArgs[$i + 1]
+      if ($tempVal -in @("true", "false")) {
+        $erlangParams += @("-logger", "handle_sasl_reports", $allArgs[++$i])
+      }
+      break
+    }
+
+    "--erl" {
+      $private:erlFlags = $allArgs[++$i] -split " "
+      $beforeExtras += $erlFlags
+      break
+    }
+
+    "+iex" {
+      $elixirParams += "+iex"
+      $useIex = $true
+
+      break
+    }
+
+    "+elixirc" {
+      $elixirParams += "+elixirc"
+      break
+    }
+
+    "--rpc-eval" {
+      $private:key = $allArgs[++$i]
+      $private:value = $allArgs[++$i]
+
+      if ($null -eq $key) {
+        Write-Error "--rpc-eval: NODE must be present"
+        exit 1
+      }
+
+      if ($null -eq $value) {
+        Write-Error "--rpc-eval: COMMAND for the '$key' node must be present"
+        exit 1
+      }
+
+      $elixirParams += "--rpc-eval"
+      $elixirParams += $key
+      $elixirParams += $value
+      break
+    }
+
+    "--boot-var" {
+      $private:key = $allArgs[++$i]
+      $private:value = $allArgs[++$i]
+
+      if ($null -eq $key) {
+        Write-Error "--boot-var: VAR must be present"
+        exit 1
+      }
+
+      if ($null -eq $value) {
+        Write-Error "--boot-var: Value for the '$key' var must be present"
+        exit 1
+      }
+
+      $elixirParams += "-boot_var"
+      $elixirParams += $key
+      $elixirParams += $value
+      break
+    }
+
+    "--pipe-to" {
+      $runErlPipe = $allArgs[++$i]
+      $runErlLog = $allArgs[++$i]
+
+      if ($null -eq $runErlPipe) {
+        Write-Error "--pipe-to: PIPEDIR must be present"
+        exit 1
+      }
+
+      if ($null -eq $runErlLog) {
+        Write-Error "--pipe-to: PIPELOG must be present"
+        exit 1
+      }
+
+      if ($runErlPipe.EndsWith("/") -or $runErlLog.EndsWith("/")) {
+        Write-Error "--pipe-to: PIPEDIR and PIPELOG must not end with a slash"
+        exit 1
+      }
+
+      break
+    }
+
+    Default {
+      $private:normalized = NormalizeArg $arg
+      $allOtherParams += $normalized
+      break
+    }
+  }
+}
+
+if ($null -eq $useIEx) {
+  $beforeExtras = @("-s", "elixir", "start_cli") + $beforeExtras
+}
+
+$beforeExtras = @("-pa", "$(Join-Path $scriptPath -ChildPath "../lib/elixir/ebin")") + $beforeExtras
+$beforeExtras = @("-noshell", "-elixir_root", "$(Join-Path $scriptPath -ChildPath "../lib")") + $beforeExtras
+
+# One MAY change ERTS_BIN= but you MUST NOT change
+# ERTS_BIN=$ERTS_BIN as it is handled by Elixir releases.
+# TODO: change when we port the releases scripts.
+# $ERTS_BIN=
```

Comment:
I don't think we will support .ps1 in releases for quite some time, so we should be fine with removing this code for now.